WalletWasabi.Fluent/ViewModels/MainViewModel.cs

```diff
@@ -99,7 +99,7 @@ public MainViewModel(UiContext uiContext)
 			? ""
 			: UiContext.ApplicationSettings.Network.Name;
 
-		// TODO: the reason why this MainViewModel singleton is even needed thoughout the codebase is dubious.
+		// TODO: the reason why this MainViewModel singleton is even needed thought the codebase is dubious.
```

Comment:
I think you meant this?

```suggestion
		// TODO: the reason why this MainViewModel singleton is even needed throughout the codebase is dubious.
```